fansubs05.txt

```diff
@@ -0,0 +1,37 @@
+Akanesasu Shoujo = simul simul simul simul
```

Comment:
whoa how did this file come back lol?

Reply:
I put the file back as an example of a fansubs file.

Reply:
oh okay so is this one of those where we take it out before merging?